src/app/api.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { HttpErrorResponse, HttpResponse } from '@angular/common/http';
+
+import { Observable, throwError } from 'rxjs';
```

Comment:
HttpResponse and Observable are not used here

Reply:
Done